### ISSUE TITLE
Support multi author

### ISSUE THREAD

**Issue by mihneadb** [mihneadb/no-nonsense#12]
- [ ] author page (`/author/author-name`)
- [ ] display author in a post
- [ ] display author in the post loop

**Comment by issmirnov** [comment]
What's your vision for this? I'd love to contribute - been using your theme for a while on my personal blog, and would love to see an "about author" section both in posts and a slug. I see from the [docs](http://themes.ghost.org/v0.5.8/docs/author) that we can reference the name/bio/site/location etc of the author.

Let me know your general direction, and how I can best help.

**Comment by mihneadb** [comment]
Great to hear that, thanks! Well, for links I'm thinking about something simple like adding the `by $Name` after the published meta:

![loop](https://cloud.githubusercontent.com/assets/643127/6137275/2b1bcd4c-b180-11e4-80bd-8b19d1179ff5.png)

![post](https://cloud.githubusercontent.com/assets/643127/6137277/2fc41e30-b180-11e4-8758-a033ed15ad01.png)

Those links should lead to the bio page of the author. Now, from what I've seen, Ghost gives you both the author's data, as well as the posts authored by them. If you have a single-author blog, the post list is redundant. However, since it allows to easily support multi-author blogs, I'd say we should keep that as well. To get to the point, I'd see the bio page simialr to a post page (check the layout of a post that also has a post photo set).

```
title: Name of the author
bio, contact info etc (whatever else is available)

smaller header: Posts by this author
{{ post loop }}
```

What do you think?

**Comment by issmirnov** [comment]
I think it definitely makes sense. Having the author link right after the publish date will keep it nice and continuous. 

For the author page, do you want to have it more of a post format, or a multicolumn expo? I know on some bio pages like [this one](http://kathyqian.com/about/) have the photo, name, and tagline on the side, with the main bio in the second column. 

Alternatively, if you want the author page more as a post, that could work, but I notice that when you have a post with a photo it's centered and substantially lower than the title, which may look odd if the user has a small (say 200x200 px) photo. The Casper theme takes this a bit too far. Ideally a combination of Crisp and Casper would probably be best, and in line with your design.

Thoughts?

**Comment by mihneadb** [comment]
Fair points.

I don't think a multicolumn layout matches the rest of the site. How about something similar to what Casper does, but without the cover photo and having everything start closer to the top?

To be fair, I am no designer, so I'd say this - feel free to experiment and post a screenshot or two here, and maybe we can decide together. What do you think? :)

**Comment by issmirnov** [comment]
Sounds good! I'll play with it over the next few days and update this post
with various screenshots.

Once again, thank you for such an awesome theme. I've converted multiple
people to Ghost after showing off my blog with it!

**Comment by mihneadb** [comment]
Thanks a lot for offering to take a shot at this! It's really great for me to have someone else contribute. :)

Haha, I'm glad to hear that!

**Comment by issmirnov** [comment]
Hey,

I added some simple one liners to add the author to the post list and post detail view. 
https://github.com/issmirnov/no-nonsense/tree/issue12-author

Screenshots: http://cl.ly/image/2v2u1w0e2Z00 and http://cl.ly/image/2f00131Q0x1q

Right now the link sends you to the author's ghost page (/author/author-name). I think this is a good default.

I'm going to twiddle the CSS for the author page tomorrow and see where it goes. Turns out ghost has some classes prepopulated for us, so it'll likely be an easy CSS injection.

**Comment by issmirnov** [comment]
Quick question - what do you use to generate assets/all.min.css? I realized my additions to style.css are ignored, seeing as all.min.css is the only file included.

**Comment by mihneadb** [comment]
Sweet!

Re: generating assets -- there's info in the readme. It should be a matter of `npm install; gulp watch` and it will automatically regen everything on reload.

**Comment by mihneadb** [comment]
BTW - you have to add the changes to the `scss` file, since the `css` file gets generated from that.

**Comment by issmirnov** [comment]
Hey - I got a fair bit of work done, here's the current code: https://github.com/issmirnov/no-nonsense/tree/issue12-author

http://cl.ly/image/423c1s210816

I'm having some issues getting the images to display properly, and mobile optimizing this whole page. 

Main issues:

- Cover photo should be much larger, similar to the post image style
- when the author has both a cover photo and a personal picture, it looks odd to have the two images stacked
- when viewing on mobile, due to hardcoding the images to specific heights and widths, various issues arise: http://cl.ly/image/3h0N1h2e1U3R

Anyway, all of my code is pushed. I would love some input on how to handle the image sizing issues, since I am a bit stuck at this time.

**Comment by mihneadb** [comment]
Hi there,

What if you use the foundation [grid classes](http://foundation.zurb.com/docs/components/grid.html) to wrap the image and set something like `width: 100%` on the `img`?

I agree, cover + personal stacked looks pretty bad. Maybe we can try personal photo only (i.e. no cover -- the theme is meant to be simple anyway).

The foundation grid (see above) should help with the mobile issue as well. And sure, if the Casper css helps, feel free to check it out. However, the `columns` classes might be easier to use.

**Comment by mihneadb** [comment]
BTW - I think the bold header style text for the bio is a bit too much, what do you think?

**Comment by issmirnov** [comment]
Didn't realize you already included foundation. Awesome! Here's a screenshot of a mobile page: http://cloud.ivansmirnov.name/image/1o1R3u3a1d0m

Overall it looks pretty decent, except for the author meta (link, location, post count) : http://cloud.ivansmirnov.name/image/2Z3V28162V3Z

I'm still trying to remember how to get Foundation to have the three divs side by side when in a large screen. and stack in mobile, without an awkward 2 + 1 stack: http://cloud.ivansmirnov.name/image/0I3X032R4040

Also, I made a clone of the post loop partial to have no author link on the title page, since it seems a tad redundant.

I'll dig around Foundation and hopefully nail the meta soon!

**Comment by issmirnov** [comment]
Grrr. Can you take a look at the foundation classes? For some reason the 3 meta spans are not stacking properly on medium sized screens: http://cloud.ivansmirnov.name/image/0T2T393Q0042

I haven't done front end in a while, so a bit at a loss.

**Comment by issmirnov** [comment]
Hey,

I styled the author page, looks good both on mobile and desktop. Added a second loop partial for the post list, so that the author page doesn't have the author links on it. Added the "by Author" to the main post list, and to the post detail page.

Screenshots:
http://cloud.ivansmirnov.name/image/1R2s1R3V3e3G
http://cloud.ivansmirnov.name/image/0m0A0b0W0L3y

What do you think? The commits are all squashed and the PR is ready.

**Comment by mihneadb** [comment]
Closed via #16. Thanks a lot @issmirnov !
